src/test/java/com/activestate/cloudfoundryjenkins/ManifestReaderTest.java

```diff
@@ -30,6 +30,19 @@ public void testGetApplicationInfo() throws Exception {
         assertEquals("hello-java", result.get("name"));
         assertEquals("512M", result.get("memory"));
         assertEquals("target/hello-java-1.0.war", result.get("path"));
+        assertEquals(null, result.get("null"));
```

Comment:
I'm not sure I understand this. The result Map doesn't actually contain a "null" key. get() will always return `null` for any non-existing key.